mitodl/open-discussions
# Add import pipeline for xPro non-program courses

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2246 

#### What's this PR do?
Imports non-program courses from xPro

#### How should this be manually tested?
- Run the xPro import on master, note how many courses you have
- Setup a new course in xpro, that isn't under a program. It should have at least one run with a product/productversion associated with it
- Run `./manage.py backpopulate_xpro_data` and verify in the UI that you see the new course and the facets and drawer work as expected
